compose.yaml

```diff
@@ -1,42 +1,35 @@
 x-env: &env
```

Comment:
I think we should also update the documentation on how to run it locally

Reply:
I've adjusted the docs 😊